### PR TITLE
Fixed unused dependencies

### PR DESCRIPTION
**Pull request by ji10drasharma**
Removed lines in package.json and yarn.lock file to remove dependencies.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 node_modules
-*.sqlite
```
Comment (ji10drasharma) [inline .gitignore:2]: no more sqlite is used so removed this line

File: package.json [hunk 2]
```diff
@@ -13,11 +13,9 @@
...
-    "mongoose": "^5.9.9",
-    "sequelize": "^5.21.6",
-    "sqlite3": "^4.1.1"
+    "mongoose": "^5.9.9"
   },
   "devDependencies": {
     "nodemon": "^2.0.3"
   }
-}
+}
```
Comment (KonradLinkowski) [inline package.json:21]: I liked that new line at the end. It helps reading files with `cat`

**Review by KonradLinkowski — CHANGES_REQUESTED** [review]
Give me back my new line :D

File: package.json (hunk 2 above)
Comment (ji10drasharma) [reply package.json:21]: okay I am on it

**Review by KonradLinkowski — APPROVED** [review]

**Comment by KonradLinkowski** [comment]
Looks cool now
Thank you for help :)

**Pull request merged**